core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -638,7 +638,8 @@ private[spark] object SparkConf extends Logging {
       DeprecatedConfig("spark.shuffle.service.index.cache.entries", "2.3.0",
         "Not used anymore. Please use spark.shuffle.service.index.cache.size"),
       DeprecatedConfig("spark.yarn.credentials.file.retention.count", "2.4.0", "Not used anymore."),
-      DeprecatedConfig("spark.yarn.credentials.file.retention.days", "2.4.0", "Not used anymore.")
+      DeprecatedConfig("spark.yarn.credentials.file.retention.days", "2.4.0", "Not used anymore."),
+      DeprecatedConfig("spark.yarn.access.namenodes", "3.0.0", "Not used anymore.")
```

Comment:
Better to keep this in the alternatives.

Reply:
Moving back. Just to learn something when can be something deprecated?

Reply:
Both lists are for deprecated configs. One if for configs that have a new name, the other for those that are just not used anymore.